include/ttmlir/Scheduler/Scheduler.h

```diff
@@ -0,0 +1,44 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TTMLIR_SCHEDULER_SCHEDULER_H
+#define TTMLIR_SCHEDULER_SCHEDULER_H
+
+#include <memory>
+
+#include "mlir/IR/Operation.h"
+
+namespace mlir::tt::scheduler {
+
+class Scheduler {
```

Comment:
We need some unit test for scheduler. Minor comments left, looks good, exactly what we need.

Reply:
Ack